src/translib/transformer/xfmr_vrf.go

```diff
@@ -514,4 +523,189 @@ var DbToYang_network_instance_route_distinguisher_field_xfmr KeyXfmrDbToYang = f
         return res_map, err
 }
 
+/* YangToDb subtree transformer for network instance interface binding */
+var YangToDb_network_instance_interface_binding_subtree_xfmr SubTreeXfmrYangToDb = func(inParams XfmrParams) (map[string]map[string]db.Value, error) {
+        var err error
+        res_map := make(map[string]map[string]db.Value)
+
+        log.Infof("YangToDb_network_instance_interface_binding_subtree_xfmr: ygRoot %v uri %v", inParams.ygRoot, inParams.uri)
+
+        pathInfo := NewPathInfo(inParams.uri)
+
+        targetUriPath, err := getYangPathFromUri(pathInfo.Path)
+
+        log.Info("YangToDb_network_instance_interface_binding_subtree_xfmr: targetUri ", targetUriPath)
+
+        /* get the name at the top network-instance table level, this is the key */
```

Comment:
This is redundant. The name validation should be done in the network-instance key transformer function. 
Is this being done got the GET case?

Reply:
removed check for "Vrf" key name.
Also removed for loop as Joyas suggested. As transformer infra is sending interface id in the key one by one to subtree transformer